+eui/MControl.m

```diff
@@ -247,10 +247,11 @@ function saveParamProfile(obj) % Called by 'Save...' button press, save a new pa
     end
     
     function loadParamProfile(obj, profile)
+      tic
```

Comment:
can the 'tic' and subsequent 'toc' be removed?

Reply:
Shit, yes.  I think there's a toc at the bottom of the script.  This was done before I wrote the performance test class